config/routes.rb

```diff
@@ -1,9 +1,29 @@
 Rails.application.routes.draw do
+  root 'welcome#index'
+
+  resources :comments
+
+  resources :answers do
+    resources :votes
+
+  end
+  resources :questions do
+    resources :votes
+
```

Comment:
I might have been tempted to nest questions under answers but make votes standalone. That way you would have urls like /questions/1-my-great-question/answers/4-some-answer. You can just pass parameters to POST /votes when you create votes.

Alternatively, if answers only ever appear as part of the question display (which I think is the case here) there's no need for any nested resources at all.